### PR TITLE
Add multiview filamat for default materials

### PR DESCRIPTION
**Pull request by z3moon**
Add prebuilt materials for the engine default materials. They'll be selected for multiview stereoscopic implementation.

### CODE REVIEW

File: libs/filamat/src/GLSLPostProcessor.cpp [hunk 1]
```diff
@@ -169,6 +169,7 @@ void GLSLPostProcessor::spirvToMsl(const SpirvBlob *spirv, std::string *outMsl,
     }
 
     mslOptions.argument_buffers = true;
+    mslOptions.ios_support_base_vertex_instance = true;
```
Comment (bejado) [inline libs/filamat/src/GLSLPostProcessor.cpp:172]: Just curious, why is this required?

**Review by bejado — APPROVED** [review]

File: libs/filamat/src/GLSLPostProcessor.cpp (hunk 1 above)
Comment (z3moon) [reply libs/filamat/src/GLSLPostProcessor.cpp:172]: Without this, the multiview shader doesn't compile for iOS, complaining `BaseInstance requires Metal 1.1 and Mac or Apple A9+ hardware` from spirv-cross.

https://github.com/KhronosGroup/SPIRV-Cross/issues/1612
